### DISCUSSION TITLE
how do i have a custom download bar in winget

### DISCUSSION THREAD

**Discussion by sastofficial** [microsoft/winget-pkgs#38682] [Q&A]
how do i have a loading bar like this? (image by @IvanovXA0)
![image](https://user-images.githubusercontent.com/39431922/147132223-9abc9fc5-c111-4aa4-91bd-b0ae024289d6.png)

**Comment by ItzLevvie** [comment]


**Reply by sastofficial** [reply]
didnt work for me https://i.imgur.com/V5qAPng.png
here is my settings file
`
    "$schema": "https://aka.ms/winget-settings.schema.json",
{"visual": {"progressBar": "rainbow"}
    // For documentation on these settings, see: https://aka.ms/winget-settings
    // "source": {
    //    "autoUpdateIntervalInMinutes": 5
    // },
}
`

**Reply by sastofficial** [reply]
thx it worked https://i.imgur.com/EfIbCtD.png
